requirements.txt

```diff
@@ -3,7 +3,7 @@ django_readonly_field
 jsonschema>=3.0
 libcomps~=0.1.15
 productmd~=1.33.0
-pulpcore>=3.13.0.dev
+pulpcore>=3.14.4
```

Comment:
Unrelated to this PR specifically, master branch actually needs `>=3.15.0.dev` because we've switched to the Django 3.2 JSONField models.